src/main/java/tech/jhipster/lite/module/domain/gradleplugin/GradleCommunityPlugin.java

```diff
@@ -44,6 +52,7 @@ public static GradleCommunityPluginIdBuilder builder() {
   private static final class GradleCommunityPluginBuilder implements GradleCommunityPluginIdBuilder, GradleCommunityPluginOptionalBuilder {
 
     private GradlePluginId id;
+    private final GradlePluginImportsBuilder<GradleCommunityPluginOptionalBuilder> imports = GradlePluginImports.builder(this);
```

Comment:
Here you can replace that with:
```suggestion
    private final Collection<GradlePluginImport> imports = new ArrayList<>();
```

and add methods
```java
    @Override
    public GradleCommunityPluginOptionalBuilder withBuildGradleImport(GradlePluginImport import) {
      imports.add(import);
      return this;
    }

    @Override
    public GradleCommunityPluginOptionalBuilder withBuildGradleImport(String import) {
      return withBuildGradleImport(new GradlePluginImport(import));
    }
```

Reply:
I have to use `gradleImport` as the parameter name because `import` got a compilation error using a reserved keyword.

Reply:
Good point! :sweat_smile: